plugins/image2/plugin.js

```diff
@@ -1697,3 +1697,13 @@ CKEDITOR.config.image2_captionedClass = 'image';
  * @cfg {String[]} [image2_alignClasses=null]
  * @member CKEDITOR.config
  */
+
+/**
+ * Determines whether alternative text is required.
+ *
+ *		config.image2_altRequired = false;
+ *
+ * @since 4.5
+ * @cfg {Boolean} [image2_prefillDimensions=false]
```

Comment:
Copy'n'pasta left-over here :) instead `image2_prefillDimensions` we want to use `image2_altRequired`.